plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -72,7 +72,7 @@ class OptunaConfig:
     # Sampling algorithm
     # Please refer to the reference for further details
     # https://optuna.readthedocs.io/en/stable/reference/samplers.html
-    sampler: Sampler = Sampler.TPESampler
+    sampler: Sampler = Sampler.tpe
```

Comment:
is the TPE sampler the recommended default by Optuna?

Reply:
Yes, it is. The default sampling algorithm is TPE as mentioned in the [reference](https://optuna.readthedocs.io/en/stable/reference/generated/optuna.study.create_study.html#optuna.study.create_study).